ksp_plugin/plugin.hpp

```diff
@@ -187,6 +187,16 @@ class Plugin {
   virtual void IncrementPartIntrinsicForce(PartId part_id,
                                            Vector<Force, World> const& force);
 
+  // Calls |MakeSingleton| for all parts in loaded vessels, enabling the use of
+  // union-find for pile up construction.  This must be called after the calls
+  // to |IncrementPartIntrinsicForce|, and before the calls to
+  // |ReportCollision|.
+  virtual void BeginReportingCollisions();
```

Comment:
The name is a bit of a misnomer, it doesn't "begin" anything.  How about `PrepareToReportCollisions`?